icons/shield_glct_lhct.svg

```diff
@@ -0,0 +1,10 @@
2,6.90539416 C14.8970417,6.74727787 15.1312625,6.98680228 15.4084296,7.1680302 C15.7400124,7.3849173 16.012307,7.35930479 16.3978745,7.34003072 C16.7342004,7.32325755 16.5753942,6.86011952 16.5610788,6.65897075 C16.5505579,6.51068552 16.5257646,6.35877832 16.605836,6.22571399 C16.7047504,6.06147484 17.0119711,6.020124 16.9996392,5.78025463 C16.9859274,5.51317737 16.6775424,5.42879406 16.4964439,5.29810126 C16.2236318,5.10122124 16.1743903,4.75907431 15.945818,4.53343412 C15.6939616,4.28476854 15.3287893,4.32392032 15.0232502,4.18848446 C14.4855599,3.95021049 14.3721577,3.46357277 13.9924112,3.10745537 C13.7929441,2.92044954 13.4993488,2.46386556 13.1971729,2.50159443 C13.0803644,2.51616855 13.0486721,2.61413425 12.9500165,2.65220807 C12.6571111,2.76526532 12.4316865,2.59339415 12.1546488,2.53833156 C11.8407447,2.47598191 11.6990135,2.5852016 11.4702687,2.7885063 C11.0219635,2.74366285 11.1372198,2.60814075 11.0061821,2.23796665 C10.9500847,2.07946229 10.3932497,2.31558033 10.3037352,2.34490105 C10.063176,2.42376516 9.78057598,2.45123178 9.53454064,2.37607587 C9.35508058,2.3203665 9.24439486,2.10524728 9.07118702,2.04902048 C8.87266849,1.98447178 8.68803419,2.10602341 8.49645778,2.13672393 C8.25594164,2.17527206 8.00675863,2.12012323 7.76809659,2.16535475 C7.5864806,2.19976348 7.41995615,2.32903336 7.231743,2.32394536 C7.11515002,2.32075457 6.97350507,2.23447404 6.85941297,2.20519644 C6.25803637,2.05104706 5.62199233,2.13603403 5.0108709,2.02241624 C4.68299626,1.96144639 4.29316003,2.03612799 3.97425408,2.11171508 C3.98628419,2.45623355 4.13926937,2.50159443 4.37616354,2.68402967 C4.54833653,2.81666282 4.59330935,2.95994628 4.69606122,3.13983752 C4.72710669,3.19425333 5.0048774,3.11870935 5.05825836,3.28001644 C5.11017328,3.43688229 4.98732809,3.46301223 4.86090404,3.54187634 C4.66587813,3.66308302 4.29510037,3.53467552 4.08890672,3.50104293 C3.85666936,3.46266728 3.67595886,3.36043282 3.44639487,3.43679605 C3.0873454,3.55623487 2.89128464,3.77838245 3.06306957,4.1219523 C3.48830625,4.97699625 4.18833845,4.26941828 4.77850419,4.61872292 C5.2243085,4.8826094 5.55696932,5.18418164 6.08215516,5.27615384 C6.67063927,5.37916443 7.00821562,5.46673852 7.26675539,6.03543114 C7.42599277,6.19466852 7.45613275,6.37016166 7.52710614,6.58579831 C7.57125969,6.72002684 7.23721907,6.73589453 7.1948765,6.82398604 C7.09626402,7.02983475 7.33634898,7.09925587 7.45324379,7.19735092 C7.6111445,7.32972535 7.71191291,7.41747192 7.79198435,7.62582151 C7.93824301,8.0061285 7.85864587,8.39208405 7.83894062,8.78437804 C7.82190874,9.12307548 7.8944344,9.4681976 7.70255616,9.77283127 C7.59795017,9.93901076 7.51766314,10.1092003 7.4922231,10.3050886 C7.47247474,10.456694 7.54241328,10.6540914 7.44992365,10.7856897 C7.28762484,11.0166766 6.85885242,10.8833104 6.69866643,11.1236541 C6.62303622,11.2372719 6.5531408,11.9404518 6.66917323,12.1688516 C7.0035588,12.2320205 7.34083333,12.2853152 7.68073809,12.3286926 L7.84644328,12.2102886 Z" fill="#FFFFFF" fill-rule="nonzero"></path>
```

Comment:
This looks so much better than https://github.com/ZeLonewolf/openstreetmap-americana/issues/569#issuecomment-1336274903.

Reply:
I imagine that icon looked better in the cartographer's software but became muddy when printed 😅